### DISCUSSION TITLE
Testing e-invoicing

### DISCUSSION THREAD

**Discussion by ZhKostev** [fattureincloud/api#84] [Q&A] [answered]
Good day!

I'm wondering what is the best way how to test sending e-invoicing in development? Does fattureincloud has special VAT & PEC for this? Also we have a question does fattureincloud have valid, but **test** VAT number for general purposes? (for IT12345678901 system will return `La Partita IVA del cliente non è valida.` ). 

Thanks in advance for the answers.

**Comment by emajo** [comment, marked as answer]
Hi @ZhKostev at the moment we don't have a test environment but a test mode for sending the e-invoices wil be released soon.
If you need test vat types you can use [this generator](https://strumentidev.it/generatore/partita-iva).

Thanks,
Emanuele.

**Reply by ZhKostev** [reply]
Thanks a lot for your answer. Got it.
